scripts/setup.sh

```diff
@@ -8,11 +8,15 @@ if [ -z "${SKIP_TERRAFORM:-}" ]; then
     terraform apply -auto-approve
 fi
 
+# for each workspace, create a new syncer
 instance_ip="$(terraform output -raw instance_ip)"
 trap "echo To debug failures, run: ssh ubuntu@$instance_ip -- journalctl -n 100 -f -u pachyderm-aml-syncer" EXIT
 
-terraform output -raw kube_config > kubeconfig
+if [ -z "${TF_VAR_existing_resource_group_name}" ]; then
```

Comment:
I didn't see this variable defined in `terraform/variables.tf`—does it need to be there?